app/models/bracket.rb

```diff
@@ -12,5 +12,36 @@ class Bracket < ApplicationRecord
   has_many :entries, dependent: :destroy
   has_many :matchups, dependent: :destroy
 
+# NEED TO MAKE AN after_create ACTION TO AUTOMATICALLY RUN THE METHODS WHEN A BRACKET IS CREATED (like in periods model of status_reporting)
```

Comment:
Whenever I leave myself code comments relating to things I still need to work on I preface them with `TODO` or `FIXME`. That way if there are a lot of them I can easily search for all the todos I need to still work on.